### PR TITLE
port parallel changes from master to 0.3.x branch

### PR DESCRIPTION
**Pull request by mikrostew**

### CODE REVIEW

File: lib/ast-plugins.js [hunk 1]
```diff
@@ -28,19 +29,30 @@ module.exports = {
...
     var templateCompilerCacheKey = fs.readFileSync(templateCompilerFullPath, { encoding: 'utf-8' });
-
-    pluginInfo.plugins.forEach(function(plugin) {
-      Compiler.registerPlugin('ast', plugin);
-    });
-
-    var PrecompileInlineHTMLBarsPlugin = HTMLBarsInlinePrecompilePlugin(Compiler.precompile, {
-      cacheKey: [templateCompilerCacheKey].concat(pluginInfo.cacheKeys).join('|')
-    });
+    var cacheKey = [templateCompilerCacheKey].concat(pluginInfo.cacheKeys).join('|');
```
Comment (stefanpenner) [inline lib/ast-plugins.js]: Thoughts on sorting this before joining. That way we ensure order of keys is always the same

Comment (mikrostew) [inline lib/ast-plugins.js]: Good idea - sorting now

**Comment by stefanpenner** [comment]
i've reviewed https://github.com/ember-cli/ember-cli-htmlbars-inline-precompile/pull/83, which this is a backport of.

**Comment by stefanpenner** [comment]
released as v0.3.13 🎉

**Pull request merged**